src/main/java/com/epam/ta/reportportal/config/DataStoreConfiguration.java

```diff
@@ -79,4 +85,15 @@ public Thumbnailator userPhotoThumbnailator(@Value("${datastore.thumbnail.avatar
 	public ContentTypeResolver contentTypeResolver() {
 		return new TikaContentTypeResolver();
 	}
+
+	private void dataStorePostInit(DataStore dataStore) {
```

Comment:
let's give it a name that will say us what this method does
"loadSaltFile" or smth

Reply:
done